SnookR/sublist/admin.py

```diff
@@ -1,3 +1,9 @@
 from django.contrib import admin
+from sublist.models import Sublist
 
-# Register your models here.
+
+class SublistAdmin(admin.ModelAdmin):
+    pass
+
```

Comment:
This particular configuration does not, we'll have to do some research on permissions.  There is absolutely a way to do model-level permissions, I am just not sure how to do this off the top of my head.